js/atree_constants.js

```diff
@@ -7093,12 +7102,18 @@ const atrees = {
                         "Spin Attack": 1
                     },
                     "display": "Total Damage"
+                },
+                {
+                    "type": "add_spell_prop",
+                    "base_spell": 1,
+                    "target_part": "Per Hit",
+                    "multipliers": [-40, 0, 0, 0, 0, 0]
```

Comment:
Due to limitations of the system, the naming between spin attack and lacerate needs to be different. Maybe keep this part named "Spin Attack" instead of "Per Hit"